tendrl/gluster_integration/sds_sync/utilization.py

```diff
@@ -40,27 +40,52 @@ def sync_utilization_details(volumes):
     out = ''
     try:
         out, err = cmd.communicate()
+
```

Comment:
Add a comment before `cmd.communicate()` call and explain the structure of `out` expected there.

Reply:
will do that.